app/services/importers/new_court_case.rb

```diff
@@ -8,7 +8,7 @@ def initialize(link)
       @case_number = link.text
       @case_types = CaseType.oscn_id_mapping
       @counties = County.pluck(:name, :id).to_h
-      @params = CGI.parse(URI(link['href']).query).map { |k, v| [k.downcase, v.first] }.to_h
+      @params = CGI.parse(URI(link['href']).query).to_h { |k, v| [k.downcase, v.first] }
```

Comment:
are you sure this works?

Reply:
I think this was a rubocop change, I don't remember doing this .